ui/src/editor/index.ts

```diff
@@ -0,0 +1,170 @@
+import {
+  type CommandProps,
+  CoreEditor,
+  Editor,
+  mergeAttributes,
+  Node,
+  Transaction,
+  ToolbarItem,
+  ToolboxItem,
+  type RawCommands,
+  type SingleCommands,
+} from "@halo-dev/richtext-editor";
+import { IconMotionLine } from "@halo-dev/components";
+import {
+  type Component,
+  type FunctionalComponent,
+  markRaw,
+  type SVGAttributes,
+} from "vue";
+import { StickerPluginKey, StickerPmPlugin } from "@/plugin";
+
+interface ExtendedRawCommands extends RawCommands {
+  openStickerPicker: () => (props: CommandProps) => boolean;
+}
+
+interface ExtendedSingleCommands extends SingleCommands {
+  openStickerPicker: (tr: Transaction) => boolean;
+}
+
+// 工具栏接口定义
+export interface ToolbarItem {
+  priority: number;
+  component: Component;
+  props: {
+    editor: Editor;
+    isActive: boolean;
+    disabled?: boolean;
+    icon?: Component;
+    title?: string;
+    action?: () => void;
+  };
+  children?: ToolbarItem[];
+}
+
+export interface ToolboxItem {
+  priority: number;
+  component: Component;
+  props: {
+    editor: Editor;
+    icon?: Component;
+    title?: string;
+    description?: string;
+    action?: () => void;
+  };
+}
+
+export interface CommandMenuItem {
+  priority: number;
+  icon: FunctionalComponent<SVGAttributes>;
+  title: string;
+  keywords: string[];
+  command: ({ editor, range }: { editor: Editor; range: Range }) => void;
+}
+
+export interface StickerOptions {
+  inline: boolean;
+  HTMLAttributes: Record<string, any>;
+  getToolbarItems: ({ editor }: { editor: Editor }) => ToolbarItem;
+  getToolboxItems: ({ editor }: { editor: Editor }) => ToolboxItem;
+  getCommandMenuItems: () => CommandMenuItem;
+}
+
+const openStickerPicker = (editor: CoreEditor) => {
+  (editor.commands as ExtendedSingleCommands).openStickerPicker(
```

Comment:
使用 `interface Commands<ReturnType>` 扩展类型后，直接使用下面的代码即可
```suggestion
  editor.commands.openStickerPicker(
```